front/src/applications/operationalStudies/components/Scenario/ScenarioDescription.tsx

```diff
@@ -22,6 +25,35 @@ const ScenarioDescription = ({
 }: ScenarioDescriptionProps) => {
   const { t } = useTranslation('operationalStudies/scenario');
   const { openModal } = useModal();
+  const [isOpenedDescription, setIsOpenedDescription] = useState<boolean>(false);
+  const descriptionRef = useRef<HTMLDivElement | null>(null);
+  const ref = useRef<HTMLDivElement | null>(null);
+  const [isTooLongDescription, setIsTooLongDescription] = useState<boolean>(false);
+
+  const toggleDescription = () => {
+    setIsOpenedDescription(!isOpenedDescription);
+  };
+
+  const checkIfTooLong = () => {
```

Comment:
Maybe add "description" in the name of this function, otherwise it's unclear what kind of thing is too long?

Reply:
Done